styles/style.css

```diff
@@ -0,0 +1,214 @@
+/* BASE RULES & OVERRIDES FOR DEFAULT BROWSER STYLES*/
+html {
+  font-size: 80%;
+  height: 100%;
+}
+
+* {
+  font-family: 'Lato', Verdana, Helvetica, sans-serif;
+  line-height: 1rem;
+  margin: 0;
+  padding: 0;
+  color: DimGrey;
+}
+
+body {
+  height: 100%;
+}
+
+.override article { padding: 1rem .5rem; margin: 1rem auto; }
+.override h1 { font-size: 4.8rem; line-height: 1.4; text-align: center; }
+.override h2 { font-size: 2.4rem; line-height: 1.4; }
+.override h3 { font-size: 2.0rem; line-height: 1.4; font-family: 'Zilla Slab', serif; padding-bottom: .25rem; }
+.override h4 { font-size: 1.6rem; line-height: 1.4; font-style: italic; }
+.override p, cite, li { font-size: 1.6rem; line-height: 1.4; padding-bottom: 1rem; }
+.override ul { text-decoration: none; }
+.override a, a:hover, a:visited, a:active, a:link { text-decoration: none; color: Teal; }
+.override a:hover { color: DarkCyan; }
+.override img { width: 100%; }
+
+/*FIXED HEADER*/
+header {
+  display: inline;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 100;
+  background-color: Coral;
+}
+
+.override header h1 {
+  float: left;
+  width: 20%;
+  margin: 0 0 0 5%;
+  padding: 2rem 0;
+  text-align: left;
+  color: DimGrey;
+}
+
+.override header a {
+  color: DimGrey;
+}
+
+header nav {
+  float: right;
+  width: 70%;
+  margin-right: 5%;
+  text-align: right;
+}
+
+#menu li {
+  font-size: 1.8rem;
+  line-height: calc((4.8rem * 1.4) + 4rem);
+  display: inline;
+  padding-left: 5%;
+}
+
+/*MAIN CONTENT AREA*/
+main {
+  display: flow-root;
+  padding: 14.6rem 10% 10rem 10%; /*for header */
+}
+
+main h3, main h4 {
+  text-align: right;
+}
+
+main p {
+  text-align: justify;
+}
+
+.intro {
+  margin-bottom: 3rem;
+}
+
+.left, .right, .double {
+  width: 100%;
+}
+
+.tint {
+  background: Teal;
+}
+
+.two-column img, .three-column img, .tint {
+  border-radius: 50%;
+}
```

Comment:
this is just a small suggestion, but:
it just happens to be coincidence that the images inside of containers with the classes `.two-column` and `.three-column` exist-- the idea of something being "two-column" and having an image inside of it isn't necessarily the most accurate representation. 

If I were to refactor this, I may make a class that's called something like "project-item" and then say an img inside of that container has a border-radius of 50%. Otherwise, you'll be stuck with having round images for every page you layout with the idea of "two-column"s etc